### PR TITLE
Turn off echo on Windows - fixes #116

### LINKED ISSUE #116: `stdout` includes prompt + command on Windows
On this line: https://github.com/GoogleChromeLabs/jsvu/blob/905885652848add5f8fe27bb909057fe777974ab/engines/chakra/test.js#L29

`stdout` is equal to:

```js
"\r\nC:\\Users\\jsvu>\"C:\\Users\\jsvu/.jsvu/engines/chakra\\chakra.exe\" \"C:\\Users\\jsvu\\AppData\\Local\\Temp\\3a4c2232c333e7ab7da85331c1dfe4e4\" \r\nHi!"
```

The expected value is only the actual stdout, which is:

```js
"Hi!"
```

In case it matters, this happens after running a batch script `chakra.cmd` with the following contents:

```cmd
"C:\Users\jsvu\.jsvu\engines\chakra\chakra.exe" %*
```

### PR DESCRIPTION
**Pull request by SamVerschueren**
This is a failing test for #116. This PR will also be used to solve the issue. See #116 for the discussion.

### CODE REVIEW

**Comment by SamVerschueren** [comment]
@sindresorhus Implemented!

// FYI @mathiasbynens

File: index.js [hunk 1]
```diff
@@ -54,6 +55,11 @@ function handleArgs(cmd, args, opts) {
 		opts.env = npmRunPath.env(Object.assign({}, opts, {cwd: opts.localDir}));
 	}
 
+	if (process.platform === 'win32' && (parsed.command === process.env.comspec || path.basename(parsed.command) === 'cmd.exe')) {
```
Comment (SamVerschueren) [inline index.js]: Is this check ok? Do we need the `parsed.command === process.env.comspec` check or is the basename always `cmd.exe`?

Comment (sindresorhus) [reply index.js]: > is the basename always cmd.exe?

Basename should always be `cmd.exe`. I think only the location could change, hence `comspec`.

File: index.js [hunk 2]
```diff
@@ -58,6 +59,11 @@ function handleArgs(cmd, args, opts) {
 		// #115
 		opts.cleanup = false;
 	}
+  
+  if (process.platform === 'win32' && (parsed.command === process.env.comspec || path.basename(parsed.command) === 'cmd.exe')) {
+		// #116
```
Comment (sindresorhus) [inline index.js]: Weird indentation

File: index.js [hunk 3]
```diff
@@ -58,6 +59,11 @@ function handleArgs(cmd, args, opts) {
 		// #115
 		opts.cleanup = false;
 	}
+  
+  if (process.platform === 'win32' && (parsed.command === process.env.comspec || path.basename(parsed.command) === 'cmd.exe')) {
```
Comment (SamVerschueren) [inline index.js]: Tried to resolve the conflict through mobile but it's a horrible UX... Feel free to fix it, otherwise will do it in the coming days.

Comment (sindresorhus) [reply index.js]: I didn't even know you could fix conflicts on mobile. I've fixed the stuff, no problem.

Comment (SamVerschueren) [reply index.js]: Yes, but the editor is really horrible :)

**Pull request merged**